WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -104,20 +107,10 @@ def index
   end
 
   def create
-    new_competition_params = params.require(:competition).permit(:name, :competition_id_to_clone)
-    @competition = Competition.new(new_competition_params.merge(
-                                   registration_open: 1.week.from_now,
-                                   registration_close: 2.weeks.from_now))
```

Comment:
Depends on why we are/were assigning arbitrary values to these fields.

Without knowing the goal it's hard to say what the best way to reach it is.